docs/retail/loyalty-rewards/setup_1_initial_streams.sql

```diff
@@ -0,0 +1,27 @@
+CREATE OR REPLACE STREAM users (
```

Comment:
@krisajenkins `or REPLACE` seems like a good practice.  Should we have ALL recipes use this?  WDYT?